packages/extension-chakra-storefront/src/utils/error-utils.js

```diff
@@ -5,14 +5,14 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import {useIntl} from 'react-intl'
-import useToast from '../../../hooks/use-toast'
-import {API_ERROR_MESSAGE} from '../../../constants'
+import useToast from '../hooks/use-toast'
+import {API_ERROR_MESSAGE} from '../constants'
 
 /**
- * Custom hook for cart error handling
+ * Custom hook for error handling
  * @returns {Function} showError function
  */
-export const useCartErrorHandler = () => {
+export const useErrorHandler = () => {
```

Comment:
This should be in hooks folder. Utils is for generic function that is not React related.

Reply:
Done